src/BuildingBlocks/EventBus/eventbus/src/main/java/com/eshoponcontainers/EventStateEnum.java

```diff
@@ -0,0 +1,16 @@
+package com.eshoponcontainers;
+
+public enum EventStateEnum {
+    NOT_PUBLISHED(0),
+    IN_PROGRESS(1),
+    PUBLISHED(2),
+    PUBLISH_FAILED(3);
+
+    private int value;
+    private EventStateEnum(int value) {
+        this.value = value;
+    }
+    public int getValue() {
```

Comment:
Added getValue for Enum as navchar2(Ex : EventStateEnum.NOT_PUBLISHED) cannot be persisted to int in db.
Need to revisit